hack/ginkgo-e2e.sh

```diff
@@ -167,15 +167,26 @@ PATH=$(dirname "${e2e_test}"):"${PATH}"
 export PATH
 
 # Choose the program to execute.
-program=("${ginkgo}")
+program=("${ginkgo}"
+         --slow-spec-threshold="${GINKGO_SLOW_SPEC_THRESHOLD:-300s}"
+         --poll-progress-after="${GINKGO_POLL_PROGRESS_AFTER:-300s}"
+         --poll-progress-interval="${GINKGO_POLL_PROGRESS_INTERVAL:-20s}"
+         --source-root="${KUBE_ROOT}"
+)
 if [[ "${E2E_TEST_DEBUG_TOOL}" == "delve" ]]; then
   program=("dlv" "exec")
 elif [[ "${E2E_TEST_DEBUG_TOOL}" == "gdb" ]]; then
   program=("gdb")
 fi
 
+# The following invocation is fairly complex. Let's dump it to simplify
+# determining what the final options are.
+set -x
+
 # NOTE: Ginkgo's default timeout has been reduced from 24h to 1h in V2, set it manually here as "24h"
-# for backward compatibility purpose.
+# for backward compatibility purpose. These (few) Ginkgo parameters are passed here
+# to ensure that they are also active when not using the Ginkgo CLI, but in general arguments
```

Comment:
How to ensure they are active when not using the Ginkgo CLI while you removing them from here?

Reply:
`{program}` could also be `dlv exec` or `gdb` I think.

Reply:
> I actually cannot get enough context of this change, which testcase will fail without this change?

Not exactly failing, but not getting reported as intended: tests are marked as "slow" by the Ginkgo CLI after the default 30s.